### PR TITLE
PAT-1890 Implement multistage Dockerfile

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
 README.md
 node_modules
 npm-debug.log
-.env*
-README.md
-*.iml
-.idea/
-.gitignore
-.git
-.circleci
```
Comment (steverendell) [inline .dockerignore:12]: tidying redundant entries and adding some new

File: Dockerfile [hunk 2]
```diff
@@ -1,45 +1,62 @@
-FROM node:12-buster-slim
```
Comment (steverendell) [inline Dockerfile:1]: Rearranged Dockerfile using pathfinder version as a guide

File: package.json [hunk 3]
```diff
@@ -69,22 +69,22 @@
   "devDependencies": {
     "cypress": "^4.12.1",
     "cypress-multi-reporters": "^1.4.0",
```
Comment (steverendell) [inline package.json:71]: Upgraded some test modules to fix a local bug with git hook

File: server/data/healthcheck.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ const superagent = require('superagent')
 const Agent = require('agentkeepalive')
 const { HttpsAgent } = require('agentkeepalive')
 const db = require('./dataAccess/db')
-const logger = require('../../log.js')
+const logger = require('../../log')
```
Comment (steverendell) [inline server/data/healthcheck.js:6]: upgraded eslint is more picky

**Review by lbennett-moj — APPROVED** [review]

**Pull request merged**